packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -11,7 +11,8 @@ const testFail = (cb, expectedDocsUrl = 'https://on.cypress.io/intercept') => {
   })
 }
 
-describe('network stubbing', { retries: { runMode: 2, openMode: 0 } }, function () {
+// TODO: Network retries leak between tests, causing flake.
+describe('network stubbing', { retries: 2 }, function () {
```

Comment:
if we dont want retries in openMode we can remove this config

Reply:
we also need to remove config from traversals_spec

Reply:
The tests are also sometimes flaky in open mode, not sure if there's a good way to fix this other than doing an `afterEach` to visit `about.blank` or something